src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy

```diff
@@ -122,6 +126,10 @@ class PartialUpdateSpec extends HealthCheckSpecification {
         given: "A flow"
         def swPair = topologyHelper.switchPairs.first()
         def flow = flowHelperV2.randomFlow(swPair)
+        flow.tap{
+            pathComputationStrategy = "cost"
+            maxLatency = 1234
+        }
```

Comment:
Why did you add it? Data was not changed. Maybe you wanted to add `maxLatencyTier2` into `data`?

Reply:
removed (test with incorrect api version)